clang/test/3C/realloc.c

```diff
@@ -15,6 +15,6 @@ void foo(int *w) {
   y[1] = 3;
   int *z = realloc(y, 5 * sizeof(int));
   //CHECK_NOALL: int *z = realloc<int>(y, 5 * sizeof(int));
-  //CHECK_ALL: _Array_ptr<int> z = realloc<int>(y, 5 * sizeof(int));
+  //CHECK_ALL: _Array_ptr<int> z : count(3 + 1) = realloc<int>(y, 5 * sizeof(int));
```

Comment:
Why doesn't `realloc` generate a bound on `z`?

Reply:
As of now, we do not insert bounds because of `realloc` (we only handle `malloc`). Lets make another issue for this.